DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/Venditore/GestioneOrdini.tex

```diff
@@ -1,12 +1,201 @@
+%	\item \textbf{Postcondizione:} il venditore ha contattato il cliente con successo;
+%	\item \textbf{Scenario principale:}
+%	\begin{itemize}
+%		\item Il venditore ha aperto il form di contatto;
+%		\item Compila il modulo apposito;
+%		\item Il venditore contatta con successo il cliente;
+%	\end{itemize}
+%\end{itemize}
+% Se si decide di mantenere questo caso d'uso va esteso meglio
+
+\UC{Filtraggio ordini nella schermata di riepilogo ordini}
+Il venditore può filtrare gli ordini nella schermata di riepilogo ordini per lo stato dell'ordine, in base al cliente che ha effettuato l'ordine, secondo un intervallo temporale ed ordinarli in base alla loro data di accettazione.
+\begin{itemize}
+	\item \textbf{Attori primari:} venditore;
+	\item \textbf{Precondizione:} il venditore è nella schermata di riepilogo ordini e ha impostato uno (o più) dei filtri disponibili per i quali cercare;
+	\item \textbf{Postcondizione:} il venditore avrà a disposizione tutti gli ordini che soddisfano tutte le condizioni dei vari filtri impostati;
+	\item \textbf{Scenario principale:} l'attore è nella schermata di riepilogo ordini e ha impostato uno (o più) dei seguenti filtri:
+	\begin{itemize}
+		\item (UC) - Impostazione del filtro per stato;
+		\item (UC) - Impostazione del filtro per intervallo temporale.
+	\end{itemize}
+	Di seguito la schermata di riepilogo ordini verrà aggiornata con gli ordini che rispettano tutti i filtri applicati;
+	%\item \textbf{Estensioni:}
+	%\begin{enumerate}[label=\lett]
+	%	\item Il venditore ha impostato i filtri in modo tale che la ricerca con la loro combinazione non dia alcun risultato. In questo caso:
+	%	\begin{itemize}
+	%		\item (UC) - Viene visualizzato il messaggio di ricerca senza alcun risultato.
+	%	\end{itemize}
+	%\end{enumerate}
+\end{itemize}
+
+\subUC{Filtro per stato}
+Il venditore può cercare gli ordini in base al loro stato, selezionando quelle di interesse tra tutti quelli disponibili.
+\begin{itemize}
+	\item \textbf{Attori primari:} venditore;
+	\item \textbf{Precondizione:} il venditore è nella schermata di riepilogo ordini e ha selezionato uno (o più) stati tra quelli disponibili per quali filtrare;
+	\item \textbf{Postcondizione:} il venditore visualizzerà nella schermata di riepilogo ordini gli ordini a suo carico che si trovano in uno degli stati selezionati;
+	\item \textbf{Scenario principale:} il venditore è nella schermata di riepilogo ordini e ha selezionato uno (o più) stati tra quelli disponibili per quali filtrare e di seguito verranno visualizzati gli ordini che appartengono ad almeno uno degli stati selezionati;
+	\item \textbf{Scenari alternativi:}
+	\begin{enumerate}[label=\lett]
+		\item Se il venditore non imposta il seguente filtro, allora verranno visualizzati tutti i prodotti a prescindere della categoria.
+	\end{enumerate}
+\end{itemize}
+
+\subUC{Filtro temporale per gli ordini effettuati}
+Il venditore filtra temporalmente l'elenco degli ordini a suo carico sulla piattaforma.
```

Comment:
Questo uc con i due sotto sono uguali a quelli per l'acquirente, li riuniamo in un unico uc e in attori mettiamo: acquirente o venditore?